src/DGmethods/DGmodel_kernels.jl

```diff
@@ -462,15 +462,23 @@ See [`odefun!`](@ref) for usage.
         l_F = MArray{Tuple{nstate}, FT}(undef)
     end
 
-    e = @index(Group, Linear)
+    _e = @index(Group, Linear)
```

Comment:
```suggestion
    eI = @index(Group, Linear) # element index
```

Reply:
Or something else, but a variable name starting with `_` is asking for a better name.

Reply:
Changed